### PR TITLE
nannari sarabath recipe

### PR DESCRIPTION
**Pull request by saikoushalg**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Can you use hyphens instead of the spaces in the filenames?

**Comment by saikoushalg** [comment]
> 
> 
> Can you use hyphens instead of the spaces in the filenames?

okay

**Comment by alfg** [comment]
Please use hyphens instead of spaces. For example:
```
nannari-sarabath.json
nannari-sarabath.jpg
```

Thanks.

**Comment by alfg** [comment]
And also remove the old files. There should only be 2 files in this PR. Thanks!

**Comment by saikoushalg** [comment]
> 
> 
> And also remove the old files. There should only be 2 files in this PR. Thanks!

okay thanks!

File: src/recipes/nannari-sarabath.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        },
+        {
+            "quantity": "2",
+            "measure": "tablespoons",
+            "ingredient": "honey"
+        }
+        
+
+
+
```
Comment (alfg) [inline src/recipes/nannari-sarabath.json:29]: Remove empty lines.

File: src/recipes/nannari-sarabath.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    "directions": [
+        " heat up the syrup and let it boil until it reaches a fine string of single thread consistency.",
+        " let it Cool down completely and add lemon juice",
```
Comment (alfg) [inline src/recipes/nannari-sarabath.json:33]: Empty leading space and capitalization, please.

File: src/recipes/nannari-sarabath.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        "Transfer to a clean container and refrigerate it",
+        "take a serving glass add 120ml of water",
+        "Add ice cubes",
+        "Give a quick stir and serve chilled."
+
```
Comment (alfg) [inline src/recipes/nannari-sarabath.json:38]: Empty line.

**Review by alfg — CHANGES_REQUESTED** [review]

**Comment by alfg** [comment]
Hmm, I'm not seeing the corrections. Can you check?

**Pull request merged**